packages/storybook/constants/globalisation.js

```diff
@@ -1,4 +1,10 @@
+export const AUSTRALIAN_LOCALE = 'en-AU';
```

Comment:
Could we export this as an object of locales? Seems a little tidier:

```suggestion
export const LOCALES = {
    au: 'en-AU',
    uk: 'en_GB',
    …
};
```

Reply:
Done.